models/localMacros.yaml

```diff
@@ -8,3 +8,8 @@ macros:
       - column
       - number_of_days
     value: " {% if warehouse.DatabaseType() == \"bigquery\" %} {% if !(end_time|isnil) %} date_diff(date('{{end_time.Format(\"2006-01-02 15:04:05\")}}'), date({{column}}), day) <={{number_of_days}} {% else %} date_diff(CURRENT_DATE(), date({{column}}), day){% endif %} <= {{number_of_days}} {% else %} {% if !(end_time|isnil) %} datediff(day, date({{column}}), date('{{end_time.Format(\"2006-01-02 15:04:05\")}}')) <= {{number_of_days}} {% else %} datediff(day, date({{column}}), GETDATE()) <= {{number_of_days}} {% endif %} {% endif %}"
+  - name: macro_listagg
+    inputs:
+      - column
+      - delimeter
+    value: " {% if warehouse.DatabaseType() == \"bigquery\" %} string_agg({{column}}, {{delimeter}}) {% else %} listagg({{column}}, {{delimeter}}) {% endif %}"
```

Comment:
{{delimeter}} should also be in single quote

Reply:
Thanks for resolving